config/db_config.yml.example

```diff
@@ -1,13 +1,13 @@
-session_secret: <%= ENV["SYSOPIA_SESSION_SECRET"] %>
-timezone_offset: <%= ENV["SYSOPIA_TIMEZONE_OFFSET"] %>
+session_secret: changeme
+timezone_offset: 0
 database:
   common: &common
     adapter: mysql2
     encoding: utf8
     reconnect: true
-    host: <%= ENV["SYSOPIA_DB_HOST"] %>
```

Comment:
According my changes we can still use ENV variables. Refer https://github.com/EOL/sysopia/blob/de0e020512b64de4999955bf2bb2abfe583e1362/config/environment.rb#L52
config.yml will be only used if ENV variables are not set.